deps/Makefile

```diff
@@ -379,8 +386,8 @@ LLVM_CPPFLAGS := $(CPPFLAGS)
 LLVM_LDFLAGS := $(LDFLAGS)
 LLVM_TARGETS := host
 LLVM_TARGET_FLAGS := --enable-targets=$(LLVM_TARGETS)
-LLVM_CMAKE += -DLLVM_TARGETS_TO_BUILD:STRING="$(LLVM_TARGETS)" -DCMAKE_BUILD_TYPE="$(LLVM_BUILDTYPE)" -DLLVM_BUILD_LLVM_DYLIB:BOOL=ON -DLLVM_DYLIB_EXPORT_ALL:BOOL=ON
-LLVM_FLAGS += --disable-profiling --enable-shared --enable-static $(LLVM_TARGET_FLAGS)
```

Comment:
at some point in the distant past, iirc, the combination of shared and static was required for building llvm on win64 (a032934d). it might be good to check that this doesn't break that target configuration